pkg/nsx/client.go

```diff
@@ -12,18 +14,26 @@ import (
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/domains"
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/domains/security_policies"
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/search"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/config"
 	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/ratelimiter"
 )
 
+var typeToString = map[ClusterHealth]string{
+	RED:    "RED",
+	ORANGE: "ORANGE",
+	GREEN:  "GREEN",
+}
```

Comment:
I think we could define constants together for cluster.go and client.go at the same place